src/systemd/sd-dhcp-server.h

```diff
@@ -48,6 +48,7 @@ int sd_dhcp_server_configure_pool(sd_dhcp_server *server, struct in_addr *addres
 int sd_dhcp_server_set_timezone(sd_dhcp_server *server, const char *timezone);
 int sd_dhcp_server_set_dns(sd_dhcp_server *server, const struct in_addr ntp[], unsigned n);
 int sd_dhcp_server_set_ntp(sd_dhcp_server *server, const struct in_addr dns[], unsigned n);
```

Comment:
This is not your fault, but could you fix the name of arguments?

Reply:
done

Reply:
you got hawk eyes for sure :)